ballot_box/views.py

```diff
@@ -171,6 +171,13 @@ def login(redirect_after=None):
         return force_auth()
 
 
+@app.route("/logout/")
+def logout():
+    response = redirect("https://registr.svobodni.cz/people/sign_out")
+    response.set_cookie("session", expires=0)  # invalidate the session
```

Comment:
To taky neni uplne idealni. To smaze cookie v prohlizeci, ale pokud ji nekdo ukradl, tak ji muze dal pouzivat.